recipe-server/client/control/components/action_fields/HeartbeatFields.js

```diff
@@ -117,6 +121,6 @@ export class HeartbeatFields extends ActionFields {
 
 export default connect(
   state => ({
-    recipeArguments: selector(state, 'arguments'),
+    recipeArguments: selector(state, 'arguments') || {},
```

Comment:
That would be good, especially for feature branches. Osmose and I ran into some trouble on the pref experiments branch because we would do unrelated fixes on the wrong branch.